src/client-side-encryption/state_machine.ts

```diff
@@ -111,7 +111,7 @@ declare module 'mongodb-client-encryption' {
  */
```

Comment:
We should add API docs explaining that secureContext will always be used if provided regardless of any other options being set.